modules/ROOT/pages/service-mesh-faqs.adoc

```diff
@@ -0,0 +1,52 @@
+= FAQs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, FAQs
+
+
+== Features Covered in Beta Release
+
+The following section addresses common questions regarding the features available in the Beta release of Anypoint Service Mesh.
+
+. I have an installation of Istio running within Kubernetes. Can I still use Anypoint Service Mesh? Is the adapter available as a standalone install?
+No, this is currently not supported. You have to uninstall the current version of Istio, and re-install Anypoint Service Mesh. 
+
+. How can I apply Istio policies for traffic control and zero trust?
+Istio policies such as fault tolerance, circuit breaker, mTLS, and so on, can be applied from Istio’s control plane using the command line. This currently is not supported in Anypoint Platform. These policies will ONLY apply to microservices deployed within the kubernetes cluster that has Anypoint Service Mesh installed. 
+
+
+. What policies can be applied within Anypoint Platform to the (non-mule) services discovered via Anypoint Service Mesh?
+The following policies can be applied to discovered non-mule services directly from Anypoint Platform:
+
+* Client ID enforcement
+* JSON Web Token (JWT)
+* Rate Limiting
+* Rate Limiting - SLA
+
+
+. What kind of monitoring data will be available within the Anypoint Platform after installing Anypoint Service Mesh?
+You can view all API Analytics available today. All the data collected by the Envoy proxy will be available in Anypoint Monitoring. 
+
+. The following components are currently NOT supported for microservices discovered with Anypoint Service Mesh:
+Application level monitoring - Services discovered are treated as HTTP APIs, so will not have application level data
+Anypoint Visualizer - Will show one node for the discovered cluster, but will not show any individual microservices within the cluster 
+
+. I use Anypoint Runtime Fabric today. Can I install Anypoint Service Mesh within my Runtime Fabric?
+No, and this is not required. Apps deployed to Anypoint Runtime Fabric are already available on Anypoint Platform. Therefore, there is no need to discover them with Anypoint Service Mesh. 
+
+. Which cloud providers are supported with Anypoint Service Mesh?
+Anypoint Service Mesh supports GKE (Google), AKS (Azure), and EKS (AWS)
```

Comment:
This needs to go in pre-requisite as well

Reply:
Done!